linux/amzn/2.0/foxpass_setup.py

```diff
@@ -147,30 +169,109 @@ def configure_sssd(bind_dn, bind_pw, backup_ldaps, opt_timeout):
     sssdconfig.save_domain(domain)
     sssdconfig.write()
 
+    if debug:
+        to_file = open_file('/etc/sssd/sssd.conf')
+        diff_files(from_file, to_file, '/etc/sssd/sssd.conf')
+
+
+def configure_ldap_sudoers(base_dn, sudo_timed, full_refresh_interval, smart_refresh_interval, debug):
+    if debug:
+        from_file = open_file('/etc/sssd/sssd.conf')
+
+    from SSSDConfig import SSSDConfig
+    sssdconfig = SSSDConfig()
+    sssdconfig.import_config('/etc/sssd/sssd.conf')
+
+    try:
+        sssdconfig.new_service('sudo')
+        sssdconfig.activate_service('sudo')
+    except:
+        pass
```

Comment:
Under what scenarios we might encounter exceptions. Should we be handling them instead of letting it proceed ?

Reply:
Resolved as discussed.